akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -179,7 +175,21 @@ object ShardRegion {
   /**
    * Reply to `GetCurrentRegions`
    */
-  @SerialVersionUID(1L) final case class CurrentRegions(regions: Set[Address]) {
+  @SerialVersionUID(2L) final case class CurrentRegions(regionActorPaths: Set[ActorPath]) {
```

Comment:
Ah, ok, I didn't understand that. So, `CurrentRegions` back to how it was, and no obvious way for the users to figure out where all regions are through this API.

So, if I understand it correctly, the summary would be something like `CurrentShardStats(stats: Seq[(ShardId, StatsAboutShard-with-enitty-count-etc)]])`?